packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.scss

```diff
@@ -0,0 +1,10 @@
+.mapsindoors-map {
```

Comment:
Can we use another selector, one that is more specific to the GoogleMapsMap component?